context_datalayer.module

```diff
@@ -55,8 +71,15 @@ function context_datalayer_context_plugins() {
  */
 function context_datalayer_context_registry() {
   return array(
+    'conditions' => array(
+      'context_datalayer_condition' => array(
+        'title' => t('Data Layer'),
+        'description' => t('React to the Data Layer using context.'),
+        'plugin' => 'context_datalayer_condition',
+      ),
+    ), 
     'reactions' => array(
-      'context_datalayer' => array(
+      'context_datalayer_reaction' => array(
```

Comment:
Context module baulks when both the reaction and the condition have the same name.  When you add a "Data Layer" reaction, the "Data Layer" options disappears from the conditions selection (and vice versa!)  This has obviously meant quite a few changes...